packages/session-replay-browser/src/remote-config-fetch.ts

```diff
@@ -103,7 +142,7 @@ export class SessionReplayRemoteConfigFetch implements AmplitudeSessionReplayRem
   completeRequest({ err, success }: { err?: string; success?: string }) {
     this.attempts = 0; // Reset attempts back to 0 for restart
     if (err) {
-      this.config.loggerProvider.warn(err);
+      throw new Error(err);
```

Comment:
I decided to change this to throw an error so that the consuming function can know that an API error occurred and behave accordingly